### DISCUSSION TITLE
Problems with two characters

### DISCUSSION THREAD

**Discussion by Lauragonz** [ffxiv-teamcraft/ffxiv-teamcraft#2890] [Q&A] [answered]
I have an account with two different characters in the game, but the app doesn't recognize the characters as separate. As a result, I see the same inventory for both characters, and the newest inventory items are not showing up correctly. Could you help me fix this?

I have already tried sorting and unsorting my inventory on both characters as suggested, but it didn't work. Additionally, I have lists from my first character showing up on the second one. I wanted to add my other character, but I wanted to fix this problem first.

**Comment by Supamiu** [comment, marked as answer]
Hi, please prefer the discord server for this, we have a bot in it that has many cool troubleshooting steps. You'll want to type `!!ichara` in the `#troubleshooting` channel in the discord server.
